src/main/java/org/dave/compactmachines3/gui/framework/widgets/WidgetList.java

```diff
@@ -71,7 +71,7 @@ public void draw(GuiScreen screen) {
         Gui.drawRect(0, 0, width, height, borderColor);
         Gui.drawRect(1, 1, width-1, height-1, backgroundColor);
 
-        //Logz.info("Rendering lines %d to %d", lineOffset, lastVisibleLine);
+        //Logz.info("Rendering lines {} to {}", lineOffset, lastVisibleLine);
```

Comment:
Prefer full removal of unnecessary logging

Reply:
Noted